### PR TITLE
Added options to tags

### PR DESCRIPTION
**Pull request by thesunwave**
Now you can write hash like this `options: { close_on_select: false }`, and use options

### CODE REVIEW

File: app/assets/javascripts/activeadmin_addons/select2.js [hunk 1]
```diff
@@ -14,10 +14,12 @@ $(function() {
     $('.select2-tags', container).each(function(i, el) {
       var model = $(el).data('model'),
           method = $(el).data('method'),
+          options = $(el).data('options'),
           selectOptions = {
             width: '80%',
             tags: $(el).data('collection')
           };
+          $.extend(selectOptions, options)
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/select2.js]: Missing semicolon.

File: app/inputs/tags_input.rb [hunk 2]
```diff
@@ -37,7 +37,8 @@ def render_tags_from_relation
 
   def tag_options
     opts = input_html_options
-    opts["data-collection"] = (@options[:collection] || []).to_json
+    opts["data-collection"] = (@options.fetch(:collection, [])).to_json
```
Comment (monkeyci) [inline app/inputs/tags_input.rb]: Don't use parentheses around a method call.

File: spec/features/select_2_spec.rb [hunk 3]
```diff
@@ -84,6 +84,25 @@ def create_items
       end
     end
 
+    context "with non empty collection includes options" do
+      before do
+        register_form(Invoice) do |f|
+          f.input :number, as: :tags, collection: ["#111", "#222", "#333"], options: { close_on_select: false }
```
Comment (monkeyci) [inline spec/features/select_2_spec.rb]: Line is too long. [111/100]

File: spec/features/select_2_spec.rb [hunk 4]
```diff
@@ -84,6 +84,28 @@ def create_items
       end
     end
 
+    context "with non empty collection includes options" do
+      before do
+        register_form(Invoice) do |f|
+          f.input :number,
+                  as: :tags,
```
Comment (monkeyci) [inline spec/features/select_2_spec.rb]: Use one level of indentation for parameters following the first line of a multi-line method call.

**Comment by ldlsegovia** [comment]
We don't want the ability to pass any option. We want to pass options that make sense with tagging functionality. That's why we separate [ajax search](https://github.com/platanus/activeadmin_addons/blob/master/docs/select2_search.md), [tags](https://github.com/platanus/activeadmin_addons/blob/master/docs/select2_tags.md) and [nested select](https://github.com/platanus/activeadmin_addons/blob/master/docs/select2_nested_select.md).
Maybe it's a good idea to unify under a select2 wrapper, but we don't plan to do this right now.

Thanks anyway

**Comment by thesunwave** [comment]
Oh, maybe I'll do it another way? I need to be able `close_on_select: false`, I think this case is found not only for me?

**Comment by kovalevsky** [comment]
@ldlsegovia if you want to add lot of tags from list, `close_on_select: false` may be extremely convenient. Look at your example: 
https://github.com/platanus/activeadmin_addons/blob/master/docs/select2_tags.md
How many actions were required to add three tags? You had to type lot of characters and make lot of clicks. This PR reduces number of steps to 4: space and 3 clicks.

Maybe not to send this property directly to Select2, but to add it support to tags?

**Pull request closed without merging**